src/windshaft/map-base.js

```diff
@@ -52,28 +59,37 @@ var WindshaftMap = Backbone.Model.extend({
   createInstance: function (options) {
     options = options || {};
 
-    var sourceLayerId = options.sourceLayerId;
-    var forceFetch = options.forceFetch;
+    var payload = this.toJSON();
+    var params = this._getParams();
 
-    var params = {
-      stat_tag: this.get('statTag')
-    };
-
-    if (this.get('apiKey')) {
-      params.api_key = this.get('apiKey');
+    var request = new Request(payload, params, options);
+    if (this._canPerformRequest(request)) {
+      this._trackRequest(request);
+      this._performRequest(request);
     }
```

Comment:
We're not doing anything when the request cannot be performed... Should we raise an error?